sdk/deviceupdate/Azure.ResourceManager.DeviceUpdate/src/Generated/Models/ErrorResponse.cs

```diff
@@ -0,0 +1,30 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+// <auto-generated/>
+
+#nullable disable
+
+using Azure.ResourceManager.Models;
+
+namespace Azure.ResourceManager.DeviceUpdate.Models
```

Comment:
It looks like that DeviceUpdate uses the common type `ErrorResponse`, I will look at why it uses this.

Reply:
It's internal